builders/interact/proxy-accounts.md

```diff
@@ -0,0 +1,181 @@
+
+The deposit is calculated based on a deposit base and a deposit factor:
+
+- **Deposit base** - the amount to be reserved for an account to have a proxy list
+- **Deposit factor** - the additional amount to be reserved for every proxy the primary account has
+
+The equation for calculating the deposit is: `deposit base + deposit factor * number of proxies`.
+
+=== "Moonriver"
+    |    Variable    |                       Value                        |
+    |:--------------:|:--------------------------------------------------:|
+    |  Deposit base  |  {{ networks.moonriver.proxy.deposit_base }} MOVR  |
+    | Deposit factor | {{ networks.moonriver.proxy.deposit_factor }} MOVR |
+    |  Max proxies   | {{ networks.moonriver.proxy.max_proxies }} proxies |
+
+=== "Moonbase Alpha"
+    |    Variable    |                       Value                        |
+    |:--------------:|:--------------------------------------------------:|
+    |  Deposit base  |   {{ networks.moonbase.proxy.deposit_base }} DEV   |
+    | Deposit factor |  {{ networks.moonbase.proxy.deposit_factor }} DEV  |
+    |  Max proxies   |  {{ networks.moonbase.proxy.max_proxies }} proxies |
+
+## Proxy Types
+
+When creating a proxy account, you must choose a type of proxy which will define how the proxy can be used. The available options are:
+
+- **`AuthorMapping`** - this type of proxy account is allowed to make transactions related to governance
+- **`CancelProxy`** - allows the proxy account to reject and remove any announced proxy calls
+- **`Staking`** - allows the proxy account to perform staking-related transactions, such as collator or delegator functions, including `authorMapping()`
+- **`Governance`** - allows the proxy account to make transactions related to governance, such as voting or proposing democracy proposals
+- **`NonTransfer`** - this type of proxy account is allowed to submit any type of transaction with the exception of balance transfers
+- **`Balances`** - allows the proxy account to only make transactions related to sending funds
+- **`Any`** - allows the proxy account to use any function supported by the proxy pallet
+
+For the purposes of this guide, you will be setting up a proxy account using the balances proxy type. 
+
+## Creating a Proxy Account
+
+There are a couple of ways you can create proxy accounts in Polkadot.js Apps, either from the **Extrinsics** page or the **Accounts** page. However, to create a time-delayed proxy, you will need to use the **Extrinsics** page. A time delay provides an additional layer of security to proxies by specifying a delay period based on a number of blocks. This will prevent the proxy account from executing a transaction until the delay period ends.
+
```

Comment:
maybe add that the delay allows time to review the pending transaction and cancel it if needed